README.md

```diff
@@ -151,6 +154,66 @@ var editor = new MediumEditor('.editable', {
+
+* __name__: name of the button being overriden
+* __action__: argument to pass to `MediumEditor.execAction()` when the button is clicked.
+* __aria__: value to add as the aria-label attribute of the button element displayed in the toolbar. This is also used as the tooltip for the button.
+* __tagNames__: array of element tag names that would indicate that this button has already been applied. If this action has already been applied, the button will be displayed as 'active' in the toolbar.
+  * _Example_: For 'bold', if the text is ever within a `<b>` or `<strong>` tag that indicates the text is already bold. So the array of tagNames for bold would be: `['b', 'strong']`
+  * __NOTE__: This is not used if `useQueryState` is set to `true`.
+* __style__: A pair of css property & value(s) that indicate that this button has already been applied. If this action has already been applied, the button will be displayed as 'active' in the toolbar.
+  * _Example_: For 'bold', if the text is ever within an element with a `'font-weight'` style property set to `700` or `'bold'`, that indicates the text is already bold.  So the style object for bold would be `{ prop: 'font-weight', value: '700|bold' }`
+  * __NOTE__: This is not used if `useQueryState` is set to `true`.
+  * Properties of the __style__ object:
+    * __prop__: name of the css property
+    * __value__: value(s) of the css property (multiple values can be separated by a `'|'`)
+* __useQueryState__: Enables/disables whether this button should use the built-in `document.queryCommandState()` method to determine whether the action has already been applied.  If the action has already been applied, the button will be displayed as 'active' in the toolbar
+  * _Example_: For 'bold', if this is set to true, the code will call `document.queryCommandState('bold')` which will return true if the browser thinks the text is already bold, and false otherwise
+* __contentDefault__: Default `innerHTML` to put inside the button
+* __contentFA__: The `innerHTML` to use for the content of the button if the __buttonLabels__ option for MediumEditor is set to `'fontawesome'`
+* __classList__: An array of classNames (strings) to be added to the button
+* __attrs__: A set of key-value pairs to add to the button as custom attributes to the button element.
+
+Example of overriding buttons (here, the goal is to mimic medium by having <kbd>H1</kbd> and <kbd>H2</kbd> buttons which actually produce `<h2>` and `<h3>` tags respectively):
+```javascript
+var editor = new MediumEditor('.editable', {
+    toolbar: {
+        buttons: [
+            'bold', 
+            'italic',
+            {
+                name: 'h1',
+                action: 'append-h2',
+                aria: 'header type 1',
+                tagNames: ['h2'],
+                contentDefault: '<b>H1</b>',
+                classList: ['custom-class-h1'],
+                attrs: {
+                    'data-header-type': 'h1'
```

Comment:
Is it only for the example or does it have any action related to this specific attribute?